R/recipe.R

```diff
@@ -436,10 +436,14 @@ prep.recipe <-
 
         # Compute anything needed for the preprocessing steps
         # then apply it to the current training set
-        x$steps[[i]] <-
-          prep(x$steps[[i]],
-               training = training,
-               info = x$term_info)
+        x$steps[[i]] <- withCallingHandlers(
+          expr = prep(x$steps[[i]], training = training, info = x$term_info),
+          error = function(c) {
+            c$call <- call(attr(x$steps[[i]], "class")[1])
+            stop(c)
```

Comment:
This should use `rlang::cnd_signal()` instead of `stop()` to preserve the reminder to call `last_error()`. With the released version of rlang this will not actually display the `call` field. However in the next version it will, see https://github.com/r-lib/rlang/pull/1235.

Reply:
Thank you!